src/AppBundle/Committee/Event/CommitteeEventFactory.php

```diff
@@ -14,6 +15,31 @@ public function __construct(PostAddressFactory $addressFactory = null)
         $this->addressFactory = $addressFactory ?: new PostAddressFactory();
     }
 
+    public function createFromArray(array $data): CommitteeEvent
+    {
+        foreach (['uuid', 'author', 'committee', 'name', 'category', 'description', 'address', 'begin_at', 'finish_at', 'capacity'] as $key) {
```

Comment:
I would like to do the same as in https://github.com/EnMarche/en-marche.fr/blob/master/src/AppBundle/Committee/CommitteeFactory.php#L20 to be consistent. What do you think?

There is many way to check the `$data` array...